### PR TITLE
Gives borg tools unused sprites

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

This PR adds pre-existing cyborg tool sprites to the cyborg tools, used in the toolset implant and cyborg inventory. It also cleans up code by moving all cyborg tools to the bottom of the list, and updates their name and description to better indicate they're twice as fast as normal tools. 
Edit: Sprites were too similar to one another, I modified them a bit.

Forgot to mention, added alpha-1 pixels to the hemostat and scalpel in the holes inside their sprite so they're easier to click too.

## Why It's Good For The Game

the sprites are really good and it's a shame they're unused. also it's hard to notice that they're actually faster than normal tools if you don't play borg/mess with cybernetic implants often

still not a real coder so pls review in case i somehow fucked up

## Changelog
:cl:
tweak: gives cyborg tools unused sprites, and changes their name and description
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/multitool.dm [hunk 1]
```diff
@@ -158,14 +158,16 @@
...
 	icon = 'icons/obj/abductor.dmi'
 	icon_state = "multitool"
 	toolspeed = 0.1
+
+/obj/item/multitool/cyborg
+	name = "electronic multitool"
+	desc = "Optimised version of a regular multitool. Streamlines processes handled by its internal microchip."
+	icon = 'icons/obj/items_cyborg.dmi'
+	icon_state = "multitool_cyborg"
+	toolspeed = 0.5
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/devices/multitool.dm]: revert the move, the only diff should be 171/172

File: code/game/objects/items/tools/crowbar.dm [hunk 2]
```diff
@@ -85,4 +77,13 @@
...
+	user.put_in_active_hand(cutjaws)
+
+/obj/item/crowbar/cyborg
+	name = "hydraulic crowbar"
+	desc = "A hydraulic prying tool, simple but powerful."
+	icon = 'icons/obj/items_cyborg.dmi'
+	icon_state = "crowbar_cyborg"
+	usesound = 'sound/items/jaws_pry.ogg'
+	force = 10
+	toolspeed = 0.5
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/tools/crowbar.dm]: same

File: code/game/objects/items/tools/weldingtool.dm [hunk 3]
```diff
@@ -376,3 +366,15 @@
...
+	name = "integrated welding tool"
+	desc = "An advanced welder designed to be used in robotic systems. Custom framework doubles the speed of welding."
+	icon = 'icons/obj/items_cyborg.dmi'
+	icon_state = "indwelder_cyborg"
+	toolspeed = 0.5
+
+/obj/item/weldingtool/largetank/cyborg/cyborg_unequip(mob/user)
+	if(!isOn())
+		return
+	switched_on(user)
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/tools/weldingtool.dm]: same

**Comment by carlarctg** [comment]
image welder
image wrench
image wirecutter
image screwdriver
image crowbar
image multitool
image knife (emag toolset)

File: code/game/objects/items/tools/wirecutters.dm [hunk 4]
```diff
@@ -124,3 +119,12 @@
...
 		..()
+
+
+/obj/item/wirecutters/cyborg
+	name = "powered wirecutters"
+	desc = "Cuts wires with the power of ELECTRICITY. Faster than normal wirecutters."
+	icon = 'icons/obj/items_cyborg.dmi'
+	icon_state = "wirecutters_cyborg"
+	toolspeed = 0.5
+	random_color = FALSE
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/tools/wirecutters.dm]: same

File: code/game/objects/items/tools/wrench.dm [hunk 5]
```diff
@@ -107,4 +102,11 @@
...
-	return OXYLOSS
\ No newline at end of file
+	return OXYLOSS
+
+/obj/item/wrench/cyborg
+	name = "hydraulic wrench"
+	desc = "An advanced robotic wrench, powered by internal hydraulics. Twice as fast as the handheld version."
+	icon = 'icons/obj/items_cyborg.dmi'
+	icon_state = "wrench_cyborg"
+	toolspeed = 0.5
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/tools/wrench.dm]: same

**Review by ExcessiveUseOfCobblestone — COMMENTED** [review]
No reason to move them, just edit the lines you want to add/remove

File: code/game/objects/items/devices/multitool.dm (hunk 1 above)
Comment (carlarctg) [reply code/game/objects/items/devices/multitool.dm]: why? i'll do it but from my pov i'm only cleaning the code up a bit and making it easier to read by standarizing a bit where the tools are in each .dm

**Comment by GunHog** [comment]
The tool sprites were set back to the old sprites because these ones were too similar to each other in shape, confusing players. https://github.com/tgstation/tgstation/pull/21870 I recommend that these be edited to be more distinct before replacing them again.

**Comment by carlarctg** [comment]
> The tool sprites were set back to the old sprites because these ones were too similar to each other in shape, confusing players. #21870 I recommend that these be edited to be more distinct before replacing them again.

goddamnit, i thought i searched enough but i guess i didn't. bleh
i guess i'll try to change the sprites myself

**Comment by Dawson1917** [comment]
The sprites are indeed too similar to eachother. I know I'd have to doublecheck a lot while selecting these as an engiborg.

**Comment by tralezab** [comment]
especially the welder and screwdriver, eesh.

**Comment by carlarctg** [comment]
new sprites by: Me
image welder
image screwdriver
image crowbar
image multitool

these actually didn't come out so bad

**Comment by tralezab** [comment]
the sprite color is much better but maybe these wouldn't be so confusing if you didn't have 3 of the 4 of those in the exact same pose

**Comment by carlarctg** [comment]
> the sprite color is much better but maybe these wouldn't be so confusing if you didn't have 3 of the 4 of those in the exact same pose

i'm not good enough of a spriter to do that much else

File: code/game/objects/items/devices/multitool.dm (hunk 1 above)
Comment (carlarctg) [reply code/game/objects/items/devices/multitool.dm]: solved with latest commit

File: code/game/objects/items/tools/weldingtool.dm (hunk 3 above)
Comment (carlarctg) [reply code/game/objects/items/tools/weldingtool.dm]: solved w/latest commit

**Comment by ExcessiveUseOfCobblestone** [comment]
the crowbar looks like something the shinobi from for honor would use as an augment.

**Comment by carlarctg** [comment]
yeah honestly i don't think i can improve the sprites any more than this. i'll leave it to the mantainers if they want it in or not

**Comment by actioninja** [comment]
If people complain they can be reverted again.

**Pull request merged**